README.md

```diff
@@ -75,13 +75,11 @@ Apart from MMDetection, we also released [MMEngine](https://github.com/open-mmla
 
 ## What's New
 
-**v3.0.0rc0** was released in 31/8/2022:
+**v3.0.0rc1** was released in 26/9/2022:
 
-- Unifies interfaces of all components based on [MMEngine](https://github.com/open-mmlab/mmengine).
-- Faster training and testing speed with complete support of mixed precision training.
-- Refactored and more flexible [architecture](https://mmdetection.readthedocs.io/en/3.x/overview.html).
-- Provides more strong baselines and a general semi-supervised object detection framework. See [tutorial of semi-supervised detection](https://mmdetection.readthedocs.io/en/3.x/user_guides/semi_det.html).
-- Allows any kind of single-stage model as an RPN in a two-stage model. See [tutorial](https://mmdetection.readthedocs.io/en/3.x/user_guides/single_stage_as_rpn.html).
+- Release [RTMDet](configs/rtmdet).
+- Refactored anchor head and base head with `boxlist`.
+- Refactored `SemiBaseDetector` and `SoftTeacher`.
```

Comment:
We can simply leave lines 80-81 and skip this.

Reply:
delete L82?

Reply:
yes